runtime-testsuite/resources/descriptors/ParserExec/ListLabelForClosureContext.txt

```diff
@@ -7,6 +7,7 @@ Parser
 
 [grammar]
 grammar T;
+@members {<setOutStream()>}
```

Comment:
i wonder if this could simply be added to output template for all tests?

Reply:
Actually it's actual only for Java runtime. It allows setting up custom output stream during function call via reflection: https://github.com/antlr/antlr4/pull/3752/files#diff-ac01582a52fe387cfa1e210df27a9f0b85a2c4470f71cb9e9701d26574455d29R153

```stg
setOutStream() ::= <<
private java.io.PrintStream outStream = System.out;

public void setOutStream(java.io.PrintStream outStream) {
	this.outStream = outStream;
}
>>
writeln(s) ::= <<outStream.println(<s>);>>
write(s) ::= <<outStream.print(<s>);>>
writeList(s) ::= <<outStream.println(<s; separator="+">);>>
```

For all other runtimes `setOutStream` is just empty, output is always being printed to console instead of custom stream.

Another way is to add support of custom output stream to runtime, but it requires changes in runtime and I'm not sure it's valuable for now.

Reply:
Maybe we use a subclass of Parser / Lexer (for java runtime testing only) that had the setOutStream() defined?

Reply:
Good idea, I'll try it.